output/tables/demographics.csv

```diff
@@ -1,5362 +1,5396 @@
-US_WV_54081,77510,,,,,,,,
-US_WV_54083,29126,,,,,,,,
-US_WV_54085,9982,,,,,,,,
-US_WV_54087,14435,,,,,,,,
-US_WV_54089,13239,,,,,,,,
-US_WV_54091,16912,,,,,,,,
-US_WV_54093,6966,,,,,,,,
-US_WV_54095,8975,,,,,,,,
-US_WV_54097,24758,,,,,,,,
-US_WV_54099,40971,,,,,,,,
-US_WV_54101,8755,,,,,,,,
-US_WV_54103,15816,,,,,,,,
-US_WV_54105,5880,,,,,,,,
-US_WV_54107,86452,,,,,,,,
-US_WV_54109,22151,,,,,,,,
-US_WY,586107,,,,,,,,
-US_WY_56001,37956,,,,,,,,
-US_WY_56003,12022,,,,,,,,
-US_WY_56005,49220,,,,,,,,
-US_WY_56007,15559,,,,,,,,
-US_WY_56009,14236,,,,,,,,
-US_WY_56011,7444,,,,,,,,
-US_WY_56013,40315,,,,,,,,
-US_WY_56015,13383,,,,,,,,
-US_WY_56017,4741,,,,,,,,
-US_WY_56019,8585,,,,,,,,
-US_WY_56021,97121,,,,,,,,
-US_WY_56023,18722,,,,,,,,
-US_WY_56025,82178,,,,,,,,
-US_WY_56027,2542,,,,,,,,
-US_WY_56029,29228,,,,,,,,
-US_WY_56031,8812,,,,,,,,
-US_WY_56033,30009,,,,,,,,
-US_WY_56035,9899,,,,,,,,
-US_WY_56037,44626,,,,,,,,
-US_WY_56039,23125,,,,,,,,
-US_WY_56041,20822,,,,,,,,
-US_WY_56043,8328,,,,,,,,
-US_WY_56045,7234,,,,,,,,
-UY,3449299,1665268,1784031,160944,3288355,1736989,1736989,19.708028,0.804
-UZ,32955400,16432433,16522967,16320173,16635227,2463969,2463969,77.469205,0.71
-VA,1000,,,,,,,,
-VC,110210,55977,54233,52683,57527,,,282.58974,0.723
-VE,28870195,14252274,14617921,3404373,25465822,2934560,9339955,32.730792,0.761
-VE_B,1440875,,,,,,,,
-VE_C,587056,,,,,,,,
-VE_D,1629433,,,,,,,,
-VE_F,2069064,,,,,,,,
-VE_G,2106264,,,,,,,,
-VE_H,500420,,,,,,,,
-VE_I,902847,,,,,,,,
-VE_J,964264,,,,,,,,
-VE_N,905443,,,,,,,,
-VE_O,426337,,,,,,,,
-VE_U,499049,,,,,,,,
-VE_V,4323467,,,,,,,,
-VE_Y,167676,,,,,,,,
-VE_Z,1422000,,,,,,,,
-VG,29802,,,15580,14222,,,198.68,
-VI,106977,50881,56096,4578,102399,,,305.64857,
-VN,95540395,47678087,47862308,61223241,34317154,8145361,16156245,308.12525,0.694
-VU,292680,148422,144258,218708,73972,,,24.009844,0.603
-WF,12197,,,,,,,,
-WS,196130,101485,94645,160350,35780,,,69.303887,0.713
-XK,1883018,,,,,,,,
-YE,28498687,14360016,14138671,18056198,10442489,2779317,2779317,53.977853,0.452
-YT,256518,,,,,,,,
-ZA,57779622,28488734,29290888,19439954,38339668,5485986,20400727,47.63012,0.699
-ZA_EC,6436761,,,,,,,,
-ZA_FS,2753200,,,,,,,,
-ZA_GP,12272263,,,,,,,,
-ZA_KZN,10267300,,,,,,,,
-ZA_LP,5404868,,,,,,,,
-ZA_MP,2065883,,,,,,,,
-ZA_NC,1145861,,,,,,,,
-ZA_NW,3509953,,,,,,,,
-ZA_WC,6844272,,,,,,,,
-ZM,17351822,8590330,8761492,9800136,7551686,2523844,2523844,23.341479,0.588
-ZW,14439018,6879217,7559801,9788355,4650663,1515016,1515016,37.324591,0.535
+key,population,population_male,population_female,population_rural,population_urban,population_largest_city,population_clustered,population_density,human_development_index,population_age_00_09,population_age_10_19,population_age_20_29,population_age_30_39,population_age_40_49,population_age_50_59,population_age_60_69,population_age_70_79,population_age_80_89,population_age_90_99
+AD,77006,58614,55571,9193,67813,,,163.84255,0.858,0,3112,2703,3644,5609,5248,3329,1812,1805,451
```

Comment:
population = 77006
population_male + population_female = 114185
What is happening here?

Reply:
I believe that the `population` field is being overridden by a different source in this pipeline: `Worldbank`. We give preference to `WorldBank` because it is less hand-wavy but in some cases it may have data up to 5 years old.

At some point, the columns are not going to match if we pull from multiple sources. For example, if we keep the `population` from `WorldPop`, then the percentage of smokers, which comes from `WorldBank`, will be off.

Reply:
Let's just make sure to clearly document where each field comes from or which fields should be consistent and which might not be, so that people trying to synthesize stats like of population that is male don't get thrown off.

Reply:
I'm also seeing that population age 0-9 is always zero (don't know why) and 10-19 for this line seems to match the male 15-19 group from metadata.csv (probably reflecting the += vs = bug...let me know when this is fixed and data regenerated and I can take another look).

Reply:
Whops, I meant the male 15-19 group from worldpop_stratified.csv

Reply:
It should be fixed now, PTAL